src/accounts/accounts.service.ts

```diff
@@ -28,7 +28,9 @@ export class AccountsService {
     const {
       polymeshService: { polymeshApi },
     } = this;
-    return await polymeshApi.accountManagement.getAccount({ address }).catch(handleSdkError);
+    return await polymeshApi.accountManagement.getAccount({ address }).catch(error => {
+      throw handleSdkError(error);
```

Comment:
I was the one who suggested to use the short form of no function declaration, but without the arrow function the "this" in handleSdk can get messed up.

There wasn't an issue before, but it could cause an unexpected bug if handleSdkError started using "this" without this change. I think its safer to use the arrow function, even if its looks like visual clutter due to JS being JS.